pkg/filesystem/subvolume.go

```diff
@@ -131,6 +139,36 @@ func getK8sRefSubvolume(ctx context.Context, clientsets *k8sutil.Clientsets) map
 	return subvolumeNames
 }
 
+// getk8sRefSnapshotHandle returns the snapshothandle for k8s ref of the volume snapshots
+func getK8sRefSnapshotHandle(ctx context.Context, clientsets *k8sutil.Clientsets) map[string]snapshotInfo {
+
+	Snap, err := snapclient.NewForConfig(clientsets.KubeConfig)
```

Comment:
Local vars generally start with lower case (camelCase). How about this var name?
```suggestion
	snapConfig, err := snapclient.NewForConfig(clientsets.KubeConfig)
```